src/components/cylc/analysis/BoxPlot.vue

```diff
@@ -241,10 +245,10 @@ export default {
         { title: 'Task name', value: 'name' },
         { title: 'Platform', value: 'platform' },
         { title: 'Count', value: 'count' },
-        { title: `Mean ${this.timingOption} time`, value: `mean${upperFirst(this.timingOption)}Time` },
-        { title: `Median ${this.timingOption} time`, value: `median${upperFirst(this.timingOption)}Time` },
-        { title: `Min ${this.timingOption} time`, value: `min${upperFirst(this.timingOption)}Time` },
-        { title: `Max ${this.timingOption} time`, value: `max${upperFirst(this.timingOption)}Time` },
+        { title: `Mean ${formatChartLabels(this.timingOption)}`, value: `mean${upperFirst(getTimingOption(this.timingOption))}` },
+        { title: `Median ${formatChartLabels(this.timingOption)}`, value: `median${upperFirst(getTimingOption(this.timingOption))}` },
```

Comment:
Not sure what is happening here but in the offline mode, sorting by median time is not working for any metric

<img width="882" height="491" alt="Image" src="https://github.com/user-attachments/assets/932c8419-6a36-4452-bb39-4e5aa47fc805" />

Possibly missing from offline data?

Reply:
Ping @ChrisPaulBennett